pwiz_tools/Skyline/SkylineNightly/Nightly.cs

```diff
@@ -382,6 +388,12 @@ public string Run()
                     Log(result = "SkylineTester has exceeded its " + durationSeconds +
                                  " second WaitForExit timeout.  You should investigate.");
                 }
+                else if (skylineTesterProcess.ExitCode == 0xDEAD)
+                {
+                    // User killed, don't post
+                    Log(result = SkylineTesterStoppedByUser);
```

Comment:
This should only mention "no results will be posted" if it is a nightly run. Otherwise, it is understood that no results will be posted, and the message should just be "SkylineTester stopped by user."